src/MultiFormGoals/resources/views/progressbar.php

```diff
@@ -32,7 +32,7 @@
         </div>
         <div part="stat-count" class="give-progress-bar-block__stat">
             <div part="stat-count-value"><?php
-                echo esc_html($this->getDonationCount()); ?></div>
+                echo esc_html($this->getFormattedDonationCount()); ?></div>
```

Comment:
I think we could simplify this by using [number_format](https://www.php.net/manual/en/function.number-format.php) directly instead of `give_format_amount`:

```suggestion
                echo esc_html(number_format($this->getDonationCount())); ?></div>
```

<img width="900" alt="Screenshot 2024-02-01 at 4 08 35 PM" src="https://github.com/impress-org/givewp/assets/10138447/b01ae781-e3c9-43f5-9345-aed3b126345f">

Reply:
Hey @DevinWalker coming back 10 months later to this lol any response? 😄

Reply:
Using `give_format_amount` still takes into account currency related settings.

![image](https://github.com/user-attachments/assets/097e1d0a-e4ae-46db-b6bf-48544d9ea612)

Reply:
I guess the question becomes, is that something we want to be taken into account to display the count/sum of is that really only useful for displaying currency amounts?

Reply:
Grouping separators are still important for non-currency amounts.

It would probably be better to use the browser locale settings, but to @DevinWalker's point the `give_format_amount` is the closest function to re-use.

Reply:
Perhaps more importantly, he is using `give_format_number` here so that it matches the other values, which are using the currency settings and not browser's locale.

So using the browser's locale would be correct in isolation, but matching the other values is probably more important in this case.

Reply:
@kjohnson I agree, in v3 forms we should be using the browser `Intl.NumberFormat` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat#using_locales

For the purposes of v2 I think this is fine 👍